src/transformers/utils/import_utils.py

```diff
@@ -1087,7 +1097,7 @@ def is_jieba_available():
         ("decord", (is_decord_available, DECORD_IMPORT_ERROR)),
         ("cython", (is_cython_available, CYTHON_IMPORT_ERROR)),
         ("jieba", (is_jieba_available, JIEBA_IMPORT_ERROR)),
-        ("peft", (is_peft_available, PEFT_IMPORT_ERROR)),
```

Comment:
We probably should not delete this

Reply:
Ah, sorry, this is a mistake during rebasing, I'll fix it!

Reply:
Fixed!